utils/symbol-rawelf.c

```diff
@@ -1,6 +1,7 @@
 #ifndef HAVE_LIBELF
 
 #include <fcntl.h>
+#include <stdbool.h>
```

Comment:
This should be added in the source code where the error is shown.
```
.../utils/symbol-rawelf.h:43:2: error: unknown type name ‘bool’
```

Reply:
Oh, the error location was in symbol-rawelf.h. Thank you for your confirmation!
I moved the stdbool.h include location to the header file.
I checked that it builds even after code changes.